libobs/obs-hotkey.h

```diff
@@ -217,6 +217,12 @@ EXPORT obs_hotkey_pair_id obs_hotkey_pair_register_source(
 	obs_hotkey_active_func func0, obs_hotkey_active_func func1, void *data0,
 	void *data1);
 
+typedef void (*obs_hotkey_released_func)(void *data, obs_hotkey_id id,
+					 obs_hotkey_t *hotkey);
+
+EXPORT void obs_hotkey_set_released_callback(obs_hotkey_id id,
+					     obs_hotkey_released_func func);
+
```

Comment:
Why do you need a separate callback when https://github.com/obsproject/obs-studio/blob/47b699e176132dbc51c03ad5a9a97a78b74ca198/libobs/obs-hotkey.h#L156-L157 is already supposed to handle both the press and release case?

Reply:
I see how it works now. If pressed is false, then the hotkey is released. I've now removed the released callback commit.